classification/feature_selection.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 28 10:33:50 2022
+
+@author: Mathilde Letard
+"""
+import numpy as np
+import pandas as pd
+import sklearn
+
```

Comment:
when the library is not in the content root of the project i'm not sure that cc_3dmasc will be found. to be tested?
from .cc_3dmasc import get_acc_expe, feature_clean
from . import cc_3dmasc